packages/shell-api/src/collection.ts

```diff
@@ -1573,4 +1568,15 @@ export default class Collection extends ShellApiClass {
     result.Totals = totalValue;
     return new CommandResult('StatsResult', result);
   }
+
+  @serverVersions(['3.1.0', ServerVersions.latest])
+  watch(pipeline: Document[] = [], options: ChangeStreamOptions = {}): ChangeStreamCursor {
+    this._emitCollectionApiCall('watch', { pipeline, options });
+    const cursor = new ChangeStreamCursor(
+      this._mongo._serviceProvider.watch(pipeline, options, {}, this._database._name, this._name),
+      this[asPrintable]()
```

Comment:
```suggestion
      this._name
```

to avoid mixing presentation with functionality?

Reply:
Sure!